include/zephyr/drivers/spi.h

```diff
@@ -891,8 +1017,17 @@ static inline int spi_write_dt(const struct spi_dt_spec *spec,
 {
 	return spi_write(spec->bus, &spec->config, tx_bufs);
 }
+/** @} */
 
 #if defined(CONFIG_SPI_ASYNC) || defined(__DOXYGEN__)
+/**
+ * @name SPI Asynchronous Transfer Functions
+ *
+ * With this API the transfer function will return immediately and
+ * report completion through a callback mechanism.
```

Comment:
A notification mechanism: a combattu or a signal

Reply:
I'm afraid I don't know what did you mean by `combattu`, do you just want to replace the word "callback" with "notification"?

Reply:
ah ah reviewing with my phone and the crappy keyboard that thinks it knows better than I do what I want to write :') 
sorry about that.

Yes: callback or signal